web/templates/page/about.html.eex

```diff
@@ -1,115 +1,82 @@
 <div class="ma3 ma4-m ma4-l">
   <h2 class="tc">About</h2>
   <div class="bb bw2 bb--hl-dark-blue pb3 pt4">
-    <p class="b">What is Healthlocker?</p>
-
+    <p class="b tc">The Greenwich integrated Children's service ADHD team</p>
     <p>
-      Healthlocker is a secure platform powered by the
-      <%= link "South London and Maudsley NHS Foundation Trust (the trust)", to: "http://www.slam.nhs.uk/",
-          class: "hl-aqua flex-wrap underline" %>
-      that promotes supported self-management and opportunities to improve
-      communication between service users, carers and clinicians.
-    </p>
-
-    <p class="b">
-      It is currently in development, more
-      features and improvements will be added in the coming months.
-    </p>
-
-    <div class="tc mv3">
-      <%= img_tag("/images/About_Clinician.png", class: "w4") %>
-    </div>
-  </div>
-
-  <div class="mt4 mb5">
-    <p class="b">Who is Healthlocker for?</p>
-
+    <p class="b">What do we do?</p>
+   <p>
+      The team provide assessment and diagnostic services for children and young people, aged 5-18, in the 
+      Greenwich borough who are presenting with symptoms of ADHD.
+   <p>
```

Comment:
There needs to be a closing `</p>` tag above to close off the previous paragraph before we open this next one.